netlogo-gui/src/main/app/CodeTabsPanel.scala

```diff
@@ -131,4 +131,11 @@ class CodeTabsPanel(workspace:            GUIWorkspace,
       new AppEvents.SwitchedTabsEvent(previousTab, currentTab).raise(this)
     }
   }
+  java.awt.EventQueue.invokeLater(new Runnable() {
+
```

Comment:
I'm not sure the blank space after the `new Runnable() {` is helpful here, I'd prefer it be between the end of the method above and the `java.awt.EventQueue` line.